src/main/java/com/pusher/rest/util/Crypto.java

```diff
@@ -0,0 +1,105 @@
+package com.pusher.rest.util;
+
+import com.google.gson.Gson;
+import org.abstractj.kalium.crypto.Random;
+import org.abstractj.kalium.crypto.SecretBox;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+/**
+ * Crypto is a class containing all the utility methods needed to do End to End Encryption for End to End Encrypted Channels.
+ * Read more here: https://pusher.com/docs/client_api_guide/client_encrypted_channels
+ */
+public class Crypto {
+    /**
+     * The prefix that is used to identify End to End Encrypted Channels
+     */
+    private static final String PRIVATE_ENCRYPTED_CHANNEL_PREFIX = "private-encrypted-";
```

Comment:
The upper case naming convention is not used anywhere in the library. I suggest everting to camel case for consistency.

Reply:
Turns out I was wrong about this. No idea why I thought this was the case. Sorry about that. I'd suggest reverting this back to `PRIVATE_ENCRYPTED_CHANNEL_PREFIX`.

I'd also suggest

* renaming `EncryptionMasterKey` to `ENCRYPTION_MASTER_KEY` here: https://github.com/pusher/pusher-http-java/blob/7ecd66c4bb49541f6d68c8147b0491ced3370a4c/src/test/java/com/pusher/rest/PusherTest.java#L30
* The same for `bodySerializer`: https://github.com/pusher/pusher-http-java/blob/7ecd66c4bb49541f6d68c8147b0491ced3370a4c/src/main/java/com/pusher/rest/Pusher.java#L57-L59

Reply:
I'll just go through the entire library and standardize on capitalized static finals

Reply:
👍